Minor corrections (#3)
fission-suite/spec: car-pool/README.md

## Motivation & High-Level Description

> This proposal resolves multiple major, persistent challenges that Fission has experienced as a production IPFS operator for the past several years. From discussion with others in the IPFS Operators Group, we believe the scope of this work to be an important stepping stone in the maturation of IPFS and Filecoin in production settings.

Reviewer: I want to reference that it's not just us, and it's happening in production AKA this is serious

Author: Yeah, I felt that you were trying to say that but the grammar was never quite right in our back and forths.  How about this.

```suggestion
This proposal resolves multiple major, persistent challenges that numerous production IPFS operators have experienced. We believe the scope of this work to be an important stepping stone in the maturation of IPFS and Filecoin in production settings.
```
